tests/integration_tests/test_automl.py

```diff
@@ -112,10 +137,39 @@ def test_data_multimodal():
 
 @pytest.mark.distributed
 @pytest.mark.parametrize(
-    "test_data",
-    ["test_data_tabular_large", "test_data_tabular_small", "test_data_image", "test_data_text", "test_data_multimodal"],
+    "test_data,expectations",
+    [
+        ("test_data_tabular_large", {"combiner": {"type": "tabnet"}}),
+        ("test_data_tabular_small", {"combiner": {"type": "concat"}}),
+        ("test_data_image", {"combiner": {"type": "concat"}}),
+        (
+            "test_data_text",
+            {
+                "input_features": [{"type": "text", "encoder": {"type": "bert"}}],
+                "combiner": {"type": "concat"},
+                "trainer": {
+                    "batch_size": "auto",
+                    "learning_rate": "auto",
```

Comment:
I'd vouch for `0.00002` instead of `auto` for LR.

Reply:
Done.